hibernate5/src/main/java/com/baeldung/hibernate/App.java

```diff
@@ -0,0 +1,31 @@
+package com.baeldung.hibernate;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import com.baeldung.hibernate.pojo.Supplier;
+
+/**
+ * Hello world!
+ *
+ */
+public class App {
+    public static void main(String[] args) {
+        try {
+            System.out.println("Checking the system.");
+            SessionFactory sessionFactory = HibernateMultiTenantUtil.getSessionFactory();
+            Session currentSession = sessionFactory.withOptions().tenantIdentifier("h2db1").openSession();
+            Transaction transaction = currentSession.getTransaction();
+            transaction.begin();
+//            currentSession.createSQLQuery("DROP ALL OBJECTS;").executeUpdate();
```

Comment:
@KevinGilmore This class is not used in article. I have just taken here for testing the boostrapping of multitenancy. Let me know if you want me to delete it.